daemon/snap.go

```diff
@@ -183,19 +179,22 @@ func mapLocal(about aboutSnap) map[string]interface{} {
 		appNames = append(appNames, appName)
 	}
 	sort.Strings(appNames)
-	apps := make([]appJSON, 0, len(localSnap.Apps))
-	for _, appName := range appNames {
+	apps := make([]*client.AppInfo, len(localSnap.Apps))
```

Comment:
We are already doing it for a few things, and I think we need to do it for more things, yes. The split is hurting us.